apps/examples/simple-example/src/simple_example/download_something.py

```diff
@@ -0,0 +1,55 @@
+"""
+Simple Example: Download Something
+-----------------------------------------
+Download image file. The PostprocessHook return the requested file as stream.
+"""
+from dataclasses import dataclass
+from pathlib import Path
+import os
+
+from hopeit.dataobjects import BinaryAttachment
+from hopeit.app.api import event_api
+from hopeit.app.logger import app_extra_logger
+from hopeit.app.context import EventContext, PostprocessHook
+
+
+__steps__ = ['find_image']
+
+
+@dataclass
```

Comment:
I think we don't need @dataclass here, since the only value contained in the class is static, not a field.

Reply:
in the rewrite of the simple example i use the class also for share information between the step and the post process.